packages/scorekeeper-status-ui/src/App.tsx

```diff
@@ -66,17 +67,17 @@ const App = () => {
           })),
         );
         setHasError(false);
-        setRefreshInterval(800); // Reset to faster refresh rate on success
+        setRefreshInterval(5000); // Reset to faster refresh rate on success
```

Comment:
hardcoded value here ?

Comment says "faster", but 5000 is the same value of the REFRESH_INTERVAL constant, which seems to be used for the "Slow down" scenario...

=> maybe we need REFRESH_INTERVAL_FAST and REFRESH_INTERVAL_SLOW ?